addons/mail/static/src/views/activity/activity_controller.scss

```diff
@@ -42,6 +42,18 @@
             margin-right: 16px;
         }
 
+        .o_m2m_avatar, .o_m2m_avatar_empty {
+            width: 32px !important;
+            height: 32px !important;
+            margin-right: 8px;
+            margin-bottom: 8px;
```

Comment:
Is it really necessary? 🤔

Reply:
Hello @xavierbol 
No, Currently not because of `col-6` in project but it'll required when `m2m_avatar` are in one column. No needed for last one. Should I move this two line for `o_m2m_avatar`  only? Thank you :slightly_smiling_face:

Reply:
I think we should find a generic design to be sure it will be good for all activity views. 🤔 
I guess the avatars are displayed in column because there is no enough place for the avatars to be displayed on a same row, right? 🤔